h2o-genmodel/src/main/java/hex/genmodel/algos/coxph/CoxPHMojoReader.java

```diff
@@ -0,0 +1,62 @@
+package hex.genmodel.algos.coxph;
+
+import hex.genmodel.ModelMojoReader;
+
+import java.io.IOException;
+import java.util.*;
+
+public class CoxPHMojoReader extends ModelMojoReader<CoxPHMojoModel> {
+
+  @Override
+  public String getModelName() {
+    return "CoxPH";
+  }
+
+  @Override
+  protected void readModelData() throws IOException {
+    _model._x_mean_cat = readRectangularDoubleArray("x_mean_cat");
+    _model._x_mean_num = readRectangularDoubleArray("x_mean_num");
+    _model._coef = readkv("coef");
+    _model._strata = readStrata();
+    _model._strata_len = readStrataLen();
+    _model._coef_indexes = readkv("coef_indexes");
```

Comment:
I don't think this is necessary, you can rely on the order of indices that you saved in the model output

imho no other algos need this

Reply:
I personally prefer to have this one explicit.

To compute indexes based on the knowledge of strata column counts, coef counts... it's possible and I can do it. But this spreads pieces of knowledge how exactly are the columns ordered to one more place. The code is not obvious and any changes will get more painful. The amount of data stored in mojo when storing `_coef_indexes` seems to be minuscule to me.

So - you are right, `_coef_indexes` are not necessary but hey lead to (IMO) better code here. But if your experience says to change this I can do it as a part of PR.

Reply:
IMHO coxph is very similar to GLM, we should follow the same pattern - it is hard for me to understand what the coef_indices actually are.

I raised a concern about handling of categorical values above - lets get back to this topic after we have clarity how categoricals are treated.